### DISCUSSION TITLE
import scale file.usd from houdini

### DISCUSSION THREAD

**Discussion by kimuss** [Autodesk/maya-usd#3012] [Q&A] [answered]
hello everyone,
I was wondering why I always scale to 100 after importing a .usd from Houdini to Maya to have the same size.
This is very annoying, because if I have a maya camera and I want to "duplicate as USD data" , I will a scale problem in houdini and the edit camera node in Houdini will be unstable.
Thank you very much for your time

**Comment by dgovil** [comment, marked as answer]
Houdini works in meters, and Maya works in centimeters. Maya doesn't do any scaling from the source USD on import or export, so it assumes all the data is centimeters and imports it at 1/100th of the scale as a result.

I have a PR [here](https://github.com/Autodesk/maya-usd/pull/2954) for export scaling out of Maya which I'm hopeful can be accepted. If it is, some of the same scaling logic could be used for importing as well in the future.
